app/helpers/validation_request_helper.rb

```diff
@@ -1,14 +1,14 @@
 # frozen_string_literal: true
 
 module ValidationRequestHelper
-  def edit_request_url(planning_application, validation_request, classname: nil)
-    link_to "Edit request",
+  def edit_request_link(planning_application, validation_request, classname: nil)
+    govuk_link_to "Edit request",
       send(:edit_planning_application_validation_validation_request_path, planning_application,
         validation_request), class: classname
   end
 
-  def delete_confirmation_request_url(planning_application, validation_request, classname: nil)
-    link_to "Delete request",
+  def delete_confirmation_request_link(planning_application, validation_request, classname: nil)
+    govuk_link_to "Delete request",
       send(:planning_application_validation_validation_request_path,
```

Comment:
Just out of interest - why the `send` here? We should be able to call the url helper directly

Reply:
It looks like we used to use string interpolation to determine which method to call, and when we stopped needing to do that the `send` was never cleaned up.